statsmodels/tsa/vecm/vecm.py

```diff
@@ -0,0 +1,2146 @@
+    sigma_u : see Parameters
+    det_coef_coint : ndarray (#(determinist. terms inside the coint. rel.) x r)
+        Estimated coefficients for the all deterministic terms inside the
+        cointegration relation.
+    const_coint : ndarray (1 x r)
+        If there is a constant deterministic term inside the cointegration
+        relation, then const_coint is the first row of det_coef_coint.
+        Otherwise it's an ndarray of zeros.
+    lin_trend_coint : ndarray (1 x r)
+        If there is a linear deterministic term inside the cointegration
+        relation, then lin_trend_coint contains the corresponding estimated
+         coefficients. As such it represents the corresponding row of
+         det_coef_coint. If there is no linear deterministic term inside the
+         cointegration relation, then lin_trend_coint is an ndarray of zeros.
+    exog_coint_coefs : ndarray (exog_coint.shape[1] x r) or None
+        If deterministic terms inside the cointegration relation are passed via
+        the exog_coint parameter, then exog_coint_coefs contains the
+        corresponding estimated coefficients. As such exog_coint_coefs
+        represents the last rows of det_coef_coint.
+        If no deterministic terms were passed via the exog_coint parameter,
+        this attribute is None.
+    det_coef : ndarray (neqs x #(deterministic terms outside the coint. rel.))
+        Estimated coefficients for the all deterministic terms outside the
+        cointegration relation.
+    const : ndarray (neqs x 1) or (neqs x 0)
+        If a constant deterministic term outside the cointegration is specified
+        within the deterministic parameter, then const is the first column of
+        det_coef_coint. Otherwise it's an ndarray of size zero.
+    seasonal : ndarray (neqs x seasons)
+        If the seasons parameter is > 0, then seasonal contains the estimated
+        coefficients corresponding to the seasonal terms. Otherwise it's an
+        ndarray of size zero.
+    lin_trend : ndarray (neqs x 1) or (neqs x 0)
+        If a linear deterministic term outside the cointegration is specified
+        within the deterministic parameter, then lin_trend contains the
+        corresponding estimated coefficients. As such it represents the
+        corresponding column of det_coef_coint. If there is no linear
+        deterministic term outside the cointegration relation, then
+        lin_trend is an ndarray of size zero.
+    exog_coefs : ndarray (neqs x exog_coefs.shape[1])
+        If deterministic terms outside the cointegration relation are passed
+        via the exog parameter, then exog_coefs contains the corresponding
+        estimated coefficients. As such exog_coefs represents the last columns
+        of det_coef.
+        If no deterministic terms were passed via the exog parameter, this
+        attribute is an ndarray of size zero.
+    delta_y_1_T : see Parameters
+    y_min1 : see Parameters
+    delta_x : see Parameters
```

Comment:
I think we make those 3 attributes private, unless we come up with good names, e.g. `_delta_y_1_T`